apps/speed.c

```diff
@@ -146,7 +146,8 @@ static void print_message(const char *s, long num, int length, int tm);
 static void pkey_print_message(const char *str, const char *str2,
                                long num, unsigned int bits, int sec);
 static void print_result(int alg, int run_no, int count, double time_used);
-#ifndef NO_FORK
+#if !defined (NO_FORK) && !defined(OPENSSL_NO_RSA) \
+    && !defined(OPENSSL_NO_DEPRECATED_3_0)
 static int do_multi(int multi, int size_num);
 #endif
```

Comment:
Er...  No.  Disabling `do_multi` is premature, as it still handles things that aren't deprecated.  Furthermore, making that a condition of `OPENSSL_NO_RSA` is wrong.

Reply:
With the RSA stuff conditioned out, it failed to compile.
I'll go through tomorrow and do something more discerning.

Reply:
Fixed this, the RSA results weren't guarded properly.  The rest of the `do_multi` bits are okay.